templates/ts-apollo-mongodb-runtime-backend/src/runtime.ts

```diff
@@ -0,0 +1,32 @@
+import { GraphbackRuntime } from 'graphback'
+import { createMongoCRUDRuntimeContext } from '@graphback/runtime-mongo'
+import { PubSub } from 'graphql-subscriptions';
+import { connectDB } from './db'
+import path from 'path';
+import { loadConfigSync } from 'graphql-config';
+
+/**
+ * Method used to create runtime schema
+ */
+export const createRuntime = async () => {
+  const db = await connectDB();
+
+  const projectConfig = loadConfigSync({
+    extensions: [
+      () => ({ name: 'graphback' }),
+      () => ({ name: 'dbmigrations' })
+    ]
+  }).getDefault()
+
+  // const projectConfig = getProjectConfig();
+  const graphbackConfig = projectConfig.extension('graphback');
+  const model = projectConfig.loadSchemaSync(path.resolve(graphbackConfig.model));
+
+  const pubSub = new PubSub();
+  const runtimeEngine = new GraphbackRuntime(model, graphbackConfig);
```

Comment:
Ah.. We require graphback config for runtime - where what we use is just crud. I think it is worth to make this small (breaking change) here to enable runtime to work without config file. 
This can be separate issue, but generally it will reduce number of steps required to create runtime

Reply:
We still need the config for:

- global CRUD configuration
- plugins

Reply:
Yep.. But why not pass it over the JS/TS. Same with plugins - as runtime do not care about them at all we can just bundle them and pass them programmatically. This will be dream come true as it will make entire runtime setup just 2 method calls (setup services and setup runtime) nothing else will be required including some strange config setup.

Reply:
It might be too much for this issue but this is great place to talk about this and explore these options as we have context of he code.